source/about/faq.rst

```diff
@@ -119,16 +116,15 @@ While this does not free us from Mojang’s control, as their code is proprietar
 modding and will continue to do so.
 
 Will I Be Able to Access the Server Internals In My Plugins?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+------------------------------------------------------------
 
 Accessing the server internals (known as "NMS" or "net.minecraft.server" in CraftBukkit) can be done through Forge,
 which has a large number of names de-obfuscated. However, be aware that accessing the server internals raises the risk
 of your plugin breaking - this is your prerogative.
 
 Will the Sponge API Be Similar to the Bukkit API in Terms of Usability?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-----------------------------------------------------------------------
 
 Yes. It will also afford you more power because you will have access to Forge.
-
 It is worth noting that the Spout API is serving as inspiration for the development of the Sponge API, which reduces the
```

Comment:
While this isn't added in this PR, it should probably be removed. While individual developers may draw their inspiration from various projects, this is (now) much too broad of a statement to be meaningful.